### PR TITLE
Some minor improvements

### PR DESCRIPTION
**Pull request by jonmv**
@bratseth please review.

### CODE REVIEW

**Review by bratseth — APPROVED** [review]
Very nice

**Comment by bratseth** [comment]
Needs ABI spec update

File: container-search/src/main/java/com/yahoo/prelude/query/CompositeItem.java [hunk 1]
```diff
@@ -205,7 +201,7 @@ protected void appendBodyString(StringBuilder buffer) {
     }
 
     /** Composite items should be parenthized when not on the top level */
-    protected boolean shouldParenthize() {
```
Comment (jonmv) [inline container-search/src/main/java/com/yahoo/prelude/query/CompositeItem.java:208]: Ah, because of this XD

**Comment by jonmv** [comment]
PTAL.

**Pull request merged**